src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -20,30 +20,109 @@
         class="mx-2"
         align="center"
       >
+        <!--        <v-col>-->
+        <!--          Get shareable link-->
+        <!--          <v-menu-->
+        <!--            offset-y-->
+        <!--            :close-on-content-click="false"-->
+        <!--            min-width="420"-->
+        <!--            max-width="420"-->
+        <!--          >-->
+        <!--            <template #activator="{ on }">-->
+        <!--              <v-icon-->
+        <!--                color="primary"-->
+        <!--                v-on="on"-->
+        <!--              >-->
+        <!--                mdi-link-->
+        <!--              </v-icon>-->
+        <!--            </template>-->
+        <!--            <v-card>-->
+        <!--              <CopyText-->
+        <!--                class="pa-2"-->
+        <!--                :text="permalink"-->
+        <!--                icon-hover-text="Copy permalink to clipboard"-->
+        <!--              />-->
+        <!--            </v-card>-->
+        <!--          </v-menu>-->
+        <!--        </v-col>-->
         <v-col>
-          Get shareable link
-          <v-menu
+          Share
+          <v-dialog
+            v-model="dialog"
             offset-y
-            :close-on-content-click="false"
             min-width="420"
-            max-width="420"
+            max-width="500"
           >
             <template #activator="{ on }">
               <v-icon
                 color="primary"
                 v-on="on"
               >
```

Comment:
Right now, this is the layout:
```
Share <v-dialog><v-icon /></v-dialog>
```
I think it might look slightly cleaner to have the "Share" text and the icon in a single button instead:
```
<v-dialog>
  <v-btn>
    Share
    <v-icon />
  </v-btn>
</v-dialog>
```
I'm not an authority on aesthetics, just a thought.

Reply:
This would make the word "share" part of the clickable area, right? I think I agree in UX terms. However, we would want to be consistent across all the buttons up there, so maybe this should be filed as an issue to address in the DLP (i.e., use the `dlp redesign` label).

Reply:
so then, I need not make the change in this PR?

Reply:
Right, no need to make changes here--the DLP handles these things via a different design already, and that should be coming to production soon anyway.